kedro_partitioned/pipeline/multinode.py

```diff
@@ -910,50 +933,10 @@ def fn(*args: Any) -> Any:
             slices, partitioneds, configurators, other_inputs = self._extract_args(args)
 
             partitioneds = self._slice_inputs(slices, partitioneds)
-
-            if self._configurator:
-                configurator_finder = ConfiguratorFinder(configurators)
-
-            outputs = [dict() for _ in range(len(self.partitioned_outputs))]
-            if partitioneds[0]:
-                for partitions in zip(
-                    *[partition.items() for partition in partitioneds]
-                ):
-                    # partitions[i][j]
-                    # i = partitioned partitions
-                    # j = key == 0, value == 1
-                    partition = get_filepath_without_extension(partitions[0][0])
-                    self._logger.info(f'Processing "{partition}" on "{self.name}"')
-
-                    configurator = []
-                    if self._configurator:
-                        possible_configurator = configurator_finder[partition]
-
-                        if possible_configurator is None:
-                            self._logger.warning(
-                                f'No configurator found for "{partition}"'
-                            )
-                        else:
-                            target = possible_configurator["target"]
-                            configurator = [possible_configurator["data"]]
-                            self._logger.info(
-                                f'Using configurator "{target}" for ' f'"{partition}"'
-                            )
-
-                    with ThreadPoolExecutor() as pool:
-                        inputs = pool.map(lambda p: p[1](), partitions)
-
-                    fn_return = self._original_func(
-                        *inputs, *configurator, *other_inputs
-                    )
-
-                    if len(self.partitioned_outputs) > 1:
-                        for i, _ in enumerate(self.partitioned_outputs):
-                            outputs[i][partition] = fn_return[i]
-                    else:
-                        outputs[0][partition] = fn_return
-
-            return outputs
+            func_return = self._original_func(*partitioneds, *other_inputs)
+            if isinstance(func_return, dict):
+                return [func_return]
+            return func_return
```

Comment:
why do we need it to return a list if it is a dict?

(honest question, I don't remember haha)